INSS.EIIR.Web/Views/SearchDetails/_DRRUDetail.cshtml

```diff
@@ -0,0 +1,61 @@
+﻿@using INSS.EIIR.Web.Constants
+@using INSS.EIIR.Web.Extensions
+@model INSS.EIIR.Models.CaseModels.CaseResult
+
+@*
+    For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+*@
+@{
+
+    <dl class="govuk-summary-list govuk-summary-list--no-border">
+
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key"> Type </dt>
+            <dd class="govuk-summary-list__value"> Debt Relief Restrictions Undertaking </dd>
+        </div>
+
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key"> Accepted date </dt>
+            <dd class="govuk-summary-list__value"> @Html.DisplayFor(model => model.restrictionsStartDate) </dd>
+        </div>
+
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key"> End date </dt>
+            <dd class="govuk-summary-list__value"> @Html.DisplayFor(model => model.restrictionsEndDate) </dd>
+        </div>
+
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key"> Duration </dt>
+            <dd class="govuk-summary-list__value"> @(Model.restrictionsStartDate.Duration(Model.restrictionsEndDate)) </dd>
+        </div>
+
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key"> NOTE </dt>
+            <dd class="govuk-summary-list__value"> This Debt Relief Restrictions Undertaking was preceded by a Debt Relief Order, details shown below </dd>
+        </div>
+
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key"> Case name </dt>
+            <dd class="govuk-summary-list__value"> @Html.DisplayFor(model => Model.caseName) </dd>
```

Comment:
If the Model.caseName is null or empty , do we need to still show the Case  name label ?
Same applies for other properties in the page

Reply:
In this instance most unlikely caseName will be null.
There are some fields which do contain nulls and here we are currently showing no text, for some but not all values.
The Legacy system does populate some text for nulls, though that is Legacy.
We are preparing screen shots to present to business of various layouts - suggest requirement for blank values be raise with them / UAT.